environment/sql/tables/binding_type.sql

```diff
@@ -1,4 +1,6 @@
 CREATE TABLE `binding_type` (
   `id` bigint(11) PRIMARY KEY AUtO_INCREMENT,
```

Comment:
Could you change this so AUTO_INCREMENT is in all upper case. You did not change this but I just noticed it while reviewing.

Reply:
went through all sql files too, was only wrong here.